pkg/store/badger_cache.go

```diff
@@ -0,0 +1,91 @@
+package store
+
+import (
+	"bytes"
+	"encoding/gob"
+	"github.com/dgraph-io/badger/v2"
+	"github.com/finitum/AAAAA/pkg/aur"
+	"github.com/pkg/errors"
+	"time"
+)
+
+type BadgerCache struct {
+	db *badger.DB
+}
+
+const userPrefix = "user_"
```

Comment:
Why is user prefix defined in the cache file and vice versa?,

Reply:
copied the wrong thing, oops